### DISCUSSION TITLE
Unable to run this command "ytmusicapi oauth"

### DISCUSSION THREAD

**Discussion by hemangkinra** [sigma67/ytmusicapi#385] [Q&A]
# Hello and thanks for using ytmusicapi! Please fill out the issue template below. Issues not following this template will be closed without comment. If you have a question only, please use GitHub discussions or gitter (linked in README).
~~~~~~~~~~~~~
**Describe the bug**
While running this command, it's giving ytmusicapi command doesn't found.
Also can someone please help me, how to generate oAuth from google api.

**To Reproduce**
Steps to reproduce the behavior:
1. I'm using macbook pro M2
2. Python version 3.10

**Additional context**
Add any other context about the problem here, such as a code sample.

**Comment by valentinfrlch** [comment]
Check if your Python is in PATH (https://www.educative.io/answers/how-to-add-python-to-the-path-variable-in-mac)
